code/game/antagonist/antagonist_add.dm

```diff
@@ -33,7 +33,7 @@
 	if(faction_verb && player.current)
 		player.current.verbs |= faction_verb
 
-	player.current?.client.verbs += /client/proc/aooc
+	player.current?.verbs += /client/proc/aooc
```

Comment:
Pretty sure player.current is not a client, so I don't know why this change is here. I would do `player.current?.client?.verbs`.

Reply:
No, it is. This code runtimes atm because it tries to access "client.client".